helm/aws-operator/templates/03-configmap.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: aws-operator-configmap
```

Comment:
I dropped `-configmap`, `-secret` and `-psp` suffixes. They are different resources.